.github/dependabot.yml

```diff
@@ -11,3 +11,9 @@ updates:
           - "> 2.1"
     commit-message:
       prefix: dependency
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
I would just make this the same as the other and add `open-pull-requests-limit: 10`

Reply:
Done